kibble/render/render.go

```diff
@@ -102,6 +103,24 @@ func Render(sourcePath string, buildPath string, cfg *models.Config) int {
 	errCount := 0
 	renderSW := utils.NewStopwatchLevel("render", logging.NOTICE)
 
+	//Replace local email logo if branding information exists for it
+	emailpath := site.SiteBrand.GetImage("email-logo", "")
```

Comment:
Would be tidier to do `emailLogoURL := site.SiteBrand["email-logo"]` here... no need to write out the extra characters with the Get function in Kibble.